styles/base.css

```diff
@@ -0,0 +1,42 @@
+/* color scheme, lightest to darkest:
+#e1e6fa
+#c4d7ed
+#abc8e2
+#375d81
+#183152
```

Comment:
Hm...you might want to look into using SASS here. We don't go over it in class, but it's not too hard to set up and would probably be very helpful.